src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1541,17 +1633,42 @@ private bool CanOpen(object parameters)
             return PathHelper.IsValidPath(filePath);
         }
 
+        private bool CanOpenFromJson(object parameters)
```

Comment:
what about reusing this helper?
https://github.com/DynamoDS/Dynamo/blob/ad0dcca2d7c2bf415f3ffcd7baff2fc06235c885/src/DynamoUtilities/PathHelper.cs#L136

Reply:
Please reuse the helper, I put a lot of thoughts into it.. Feel free to update as well

Reply:
Done